Types/Entities/Sprite.js

```diff
@@ -98,7 +109,29 @@ function Sprite(spec = {}) {
   }
 
   function draw() {
-    camera.drawThrough(ctx, drawLocal, transform)
+    // If it is part of the background, it is transformed
+    if (fixed) {
+      if (sky.height / sky.width >= screen.height / screen.width) {
+        let adjustmentY = (screen.width * sky.height) / sky.width
+        pos.set(factorX*screen.width, factorY*(adjustmentY) + (screen.height - adjustmentY)/2)
+        sizer.set(screen.width/sky.width, screen.width/sky.width)
+      } else {
+        let adjustmentX = (screen.height * sky.width) / sky.height
+        pos.set(factorX*(adjustmentX)+(screen.width-adjustmentX)/2, factorY*screen.height)
+        sizer.set(screen.height/sky.height, screen.height/sky.height)
+      }
+
+      // Also it needs to be rotated differently also
+      if (rotatingSpeed !=0){
```

Comment:
Just get rid of this wrapping `if` statement, it's not needed (and right now if rotatingSpeed were 0, the sprite simply would not draw)